netlib/websockets/frame.py

```diff
@@ -128,17 +141,16 @@ def __bytes__(self):
             # '!Q' = pack as 64 bit unsigned long long
             # add 8 bytes extended payload length
             b += struct.pack('!Q', self.payload_length)
+        else:
+            raise ValueError("Payload length exceeds 64bit integer")
```

Comment:
No, not `bytes`, but the only argument is invalid. I wasn't sure what to use - RuntimeError seemed too generic.

> exception ValueError
Raised when a built-in operation or function receives an argument that has the right type but an inappropriate value, and the situation is not described by a more precise exception such as IndexError.

vs. 
> exception RuntimeError
Raised when an error is detected that doesn’t fall in any of the other categories. The associated value is a string indicating what precisely went wrong.